internal/cephfs/snapshot.go

```diff
@@ -238,3 +238,40 @@ func cloneSnapshot(ctx context.Context, parentVolOptions *volumeOptions, cr *uti
 	}
```

Comment:
commenting on the commit message:

    subvolume firts. This commits adds

firts -> first
commits -> commit